src/components/Markers/SpotMarker.tsx

```diff
@@ -1,13 +1,43 @@
-import { Location } from "datasource";
+import classNames from "classnames";
+import { Icon } from "components/Icon";
+import { Container, Location } from "datasource";
+import { divIcon, DivIcon } from "leaflet";
 import React from "react";
+import { renderToStaticMarkup } from "react-dom/server";
 import { Marker } from "react-leaflet";
+import { getContainerColor } from "utils/container";
 
 type Props = {
   location: Location;
+  containers: Container[];
 };
 
 export const SpotMarker: React.FC<Props> = props => {
+  const SpotIcon: DivIcon = divIcon({
+    html: renderToStaticMarkup(
```

Comment:
add comment: `Leaflet requires plain html in divIcon, renderToStaticMarkup returns string with plain html`